server/remote/bitbucket/fixtures/handler.go

```diff
@@ -27,6 +27,7 @@ func Handler() http.Handler {
 	gin.SetMode(gin.TestMode)
 
 	e := gin.New()
+	e.GET("/2.0/workspaces/", getWorkspace)
```

Comment:
As of https://developer.atlassian.com/cloud/bitbucket/bitbucket-api-teams-deprecation/ this new path should completely replace the "/2.0/teams/" path below which doesn't exist anymore.

Reply:
Then jut delete the old one ...